### ISSUE TITLE
pypi does not render README

### ISSUE THREAD

**Issue by fehiepsi** [pyro-ppl/numpyro#184]
Current [pypi description](https://pypi.org/project/numpyro/) does not show README properly. Looking like we get the same issue at [this thread](https://stackoverflow.com/questions/26737222/pypi-description-markdown-doesnt-work). This happens in Pyro too. I think that we need to update `setuptools`, `twine`, `wheel` (or pinning to some working versions).

**Comment by neerajprad** [comment]
I realized this after uploading that the legacy command `python setup.py sdist upload` does not seem to render markdown, and we need to use twine to upload. Unfortunately, updating the description requires uploading a new release, so we will have to wait for the next one to fix this.

Pyro's README is actually fixed by @fritzo which works even with the legacy commands because it converts markdown to rst (and removes badges). Unfortunately, I didn't have `pypandoc` installed in my particular environment when I uploaded the last release and it didn't convert it into `rst`, but this should be fixed too with the next release. 

If markdown doesn't work for some reason, I think we can just use the tried and tested way of converting to rst as in Pyro. Let us keep the issue open until that is fixed.
